### PR TITLE
Adds the two new software from 0.10.0 B1 to the editor.

### PR DESCRIPTION
**Pull request by smk4664**
This also bumps the version to 1.57

<details>
<summary>Spoilers</summary>
Adds Software.InfinityHorizon and Software.WaveRestart.

</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
             </div>
 
             <div class="editor-info">
-                <span class="infotext">Editor: 1.56 ─ Game: v0.10.0 B1</span>
+                <span class="infotext">Editor: 1.57 ─ Game: v0.10.0 B1</span>
```
Comment (d0sboots) [inline index.html]: Mentioned in Discord: Please revert this part, I keep the version in sync with upstream (Kyromyr)

File: _config.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```
Comment (d0sboots) [inline _config.yml]: What's this for?

Comment (smk4664) [reply _config.yml]: Whoops, I was testing the github pages, will revert

File: index.html (hunk 1 above)
Comment (smk4664) [reply index.html]: I also entered a PR in the upstream, but I can revert.

**Pull request merged**